lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -467,10 +468,11 @@ export class PublicClientApplication {
      * Default behaviour is to redirect the user to `window.location.href`.
      * @param logoutRequest 
      */
-    logout(account: AccountInfo, postLogoutRedirectUri?: string, authority?: string): void {
-        this.createAuthCodeClient(authority).then((authClient: AuthorizationCodeClient) => {
+    logout(logoutRequest: EndSessionRequest): void {
```

Comment:
Thoughts on making this `async`? This seems like a good time, as you are changing the function signature.

Reply:
done.